sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py

```diff
@@ -534,13 +531,13 @@ async def add_participants(
     @distributed_trace_async
     async def remove_participant(
         self,
-        user: CommunicationUserIdentifier,
+        user: CommunicationIdentifier,
```

Comment:
This argument also needs to be renamed to `identifier` @juancamilor @sarkar-rajarshi